### PR TITLE
Task 1

### PR DESCRIPTION
**Pull request by BM-Fisher**
Still, need to implement styling.

### CODE REVIEW

**Comment by BM-Fisher** [comment]
Using the same pull request throught project since it is still open

File: instagram-app/src/components/CommentSection/CommentSection.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+        addComment = event => {
+        event.preventDefault();
+        const newerComment = {
+            id: Date.now(),
+            username: "Brittany is amazing...always",
+            text: this.state.newComment
+        }
+        this.setState({
+          comment: [...this.state.comment, newerComment], newComment:""
```
Comment (LeTanque) [inline instagram-app/src/components/CommentSection/CommentSection.js]: Nice spread operator!

File: instagram-app/src/components/PostContainer/Like.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+class Like extends React.Component {
+    constructor(props){
+        super(props);
+        this.state={
+            likes: props.likeData,
+            liked: false
+        }
+    }
+
+        addLike = bananas => {
```
Comment (LeTanque) [inline instagram-app/src/components/PostContainer/Like.js]: I love the literal bananas!

File: instagram-app/src/components/PostContainer/Like.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+                });
+            }
+
+    }
+    render(){
+        return(
+            <>
+            <div className="like-icons">
+                <FiHeart className="heart" onClick={this.addLike}/>
+                <FiMessageCircle className="circle"/>
```
Comment (LeTanque) [inline instagram-app/src/components/PostContainer/Like.js]: Good job using react-icons

File: instagram-app/src/components/PostContainer/PostContainer.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+import "./Container.scss";
+import Like from "./Like.js";
+import CommentSection from "../CommentSection/CommentSection.js"
+
+
+const PostContainer = (props) => {
+    console.log(props);
+    return(
+        <>
+        {/* map doesn't change the original array forEach would. Map lets you choose what display*/}
```
Comment (LeTanque) [inline instagram-app/src/components/PostContainer/PostContainer.js]: I like how you are commenting your code like notes. This is a very helpful practice, IMO

**Pull request merged**